pkg/resource/db_cluster/custom_update.go

```diff
@@ -23,6 +23,7 @@ import (
 	svcsdk "github.com/aws/aws-sdk-go/service/rds"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	 "slices"
```

Comment:
nit: Please order the import according to the go standards: https://go.dev/wiki/CodeReviewComments#imports